I get:
````
Error:
/tmp/nix-build-pony-stable-2016-10-10.drv-0/pony-stable-fdefa26fed93f4ff81c323f29abd47813c515703-src/stable/main.pony:67:9: argument not a subtype of parameter
        ["env", "PONYPATH="+ponypath].append(rest), env~exitcode()
        ^
    Info:
    /nix/store/zc000mvg5vvwsgm4m3vidfnlj6c2w3sc-ponyc-0.11.1/lib/pony/0.11.1/packages/builtin/array.pony:266:3: None val is not a subtype of Array[String val] box
      fun ref append(seq: (ReadSeq[A] & ReadElement[A^]), offset: USize = 0,
      ^
Error:
/tmp/nix-build-pony-stable-2016-10-10.drv-0/pony-stable-fdefa26fed93f4ff81c323f29abd47813c515703-src/stable/shell.pony:30:30: argument not a subtype of parameter
    "'" + arg.clone().replace("'", "'\\''") + "'"
                             ^
    Info:
    /nix/store/zc000mvg5vvwsgm4m3vidfnlj6c2w3sc-ponyc-0.11.1/lib/pony/0.11.1/packages/builtin/string.pony:928:68: USize val is not a subtype of String box
      fun ref replace(from: String box, to: String box, n: USize = 0): USize =>
                                                                       ^
make: *** [Makefile:15: bin/stable] Error 255
builder for ‘/nix/store/81qyaxgkbx73grwvp582la7dbjb3s64h-pony-stable-2016-10-10.drv’ failed with exit code 2
````

--- a/pkgs/development/compilers/ponyc/default.nix
+++ b/pkgs/development/compilers/ponyc/default.nix
@@ -3,13 +3,13 @@
 
 stdenv.mkDerivation ( rec {
   name = "ponyc-${version}";
-  version = "0.11.0";
+  version = "0.11.3";
 
   src = fetchFromGitHub {
     owner = "ponylang";
     repo = "ponyc";
     rev = version;
-    sha256 = "0jjzc1711km53zkkxwy4r9ki9yxdx8mdjrp7g3mkr42p7q5h4w70";
+    sha256 = "08wh8rh17bf043glvn7778bwpxyzpm95kgfll645hf2m65n5ncsh";
   };
 
   buildInputs = [ llvm makeWrapper which ];

--- a/pkgs/development/compilers/ponyc/pony-stable.nix
+++ b/pkgs/development/compilers/ponyc/pony-stable.nix
@@ -1,13 +1,13 @@
 {stdenv, fetchFromGitHub, ponyc }:
 
 stdenv.mkDerivation {
-  name = "pony-stable-2016-10-10";
+  name = "pony-stable-unstable-2017-01-03";
 
   src = fetchFromGitHub {
     owner = "jemc";
     repo = "pony-stable";
-    rev = "fdefa26fed93f4ff81c323f29abd47813c515703";
-    sha256 = "16inavy697icgryyvn9gcylgh639xxs7lnbrqdzcryvh0ck15qxk";
+    rev = "0054b429a54818d187100ed40f5525ec7931b31b";
+    sha256 = "0libx8byzwqjjgxxyiiahiprzzp845xgbk09sx9bzban5cd5j0g5";
   };
 
   buildInputs = [ ponyc ];

